scripts/algolia/transform-mdx-to-algolia-record/create-records.mjs

```diff
@@ -55,13 +55,16 @@ export async function createAlgoliaRecordObject(
 	const codeListItems = await collectCodeListItems(markdownFile)
 
 	return {
-		objectID,
-		page_title: frontmatter.page_title,
-		type: 'docs',
-		products: [productSlug],
-		description: frontmatter.description ?? '',
-		headings,
-		codeListItems,
-		_tags: [directory],
+		action: 'addObject',
```

Comment:
Needed for batch requests. See docs [here](https://www.algolia.com/doc/libraries/javascript/v5/methods/search/batch/?client=javascript)